subworkflows/local/bam_variant_calling_smoove/main.nf

```diff
@@ -2,10 +2,11 @@
 // Run Delly
```

Comment:
why does it say Delly here?

Reply:
Probably a copy paste error from way back in the day :grin: I'll remove it! Thanks for spotting!